ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+[](Github issues should be used only for bugs and feature requests. For questions please visit stackoverflow.com/tags/onsen-ui or community.onsen.io. For bugs please use the following template.)
```

Comment:
That was the intent. It's intended to be a comment. I think what is written here is something which would be really nice if the users see before adding the issue.